webui/src/Views/InstanceView.elm

```diff
@@ -441,23 +437,28 @@ instanceAllocationRow instance index ( taskName, allocation ) =
                     ( "running", "label-success" )
     in
         tr []
-            [ td [] [ code [] [ text (shortAllocationId allocation.id) ] ]
+            [ td [] [ code [] [ text (shortAllocationId task.allocationId) ] ]
             , td [ class "text-center" ]
                 [ span [ class ("label " ++ labelKind) ] [ text description ]
                 ]
-            , td [] [ text taskName ]
+            , td [] [ text task.taskName ]
+            , td [ class "text-center" ]
+                [ text (unwrap "unknown" (Round.round 2) task.cpuTicksUsed)
```

Comment:
We should also add the unit here (MHz).

Reply:
I'm not sure these are MHz.  Nomad exposes them as CPU ticks, in my understanding that's not the same as MHz.

Reply:
It corresponds to CPU shares (MHz) and we are going to put it into relation to the allocated CPU shares (MHz) in order to get the percentage.

```
Task Resources
CPU         Memory          Disk     IOPS  Addresses
33/100 MHz  18 MiB/128 MiB  300 MiB  0     ui: 127.0.0.1:8000
```

![image](https://user-images.githubusercontent.com/3427394/29869130-f7648396-8d81-11e7-8263-9da03104dde0.png)

- https://www.nomadproject.io/docs/agent/telemetry.html#host-metrics

Reply:
@FRosner I was confused by the name “ticks“ to be honest.  In Unix "tick" normally refers to the internal timer of the kernel not to CPU clock speed.